notifications_api_common/admin.py

```diff
@@ -1,4 +1,6 @@
+from django.conf import settings
 from django.contrib import admin, messages
+from django.core.exceptions import ValidationError
```

Comment:
If you test this in the browser, this currently results in a 500 error, instead of properly displaying the error to the user, I think you have to use `from django.forms import ValidationError` instead

<img width="1183" height="163" alt="image" src="https://github.com/user-attachments/assets/931dbad5-22ec-4b51-84f7-916c82c5ef9b" />

Could you also add a test for this?